presentation/src/main/java/com/threegap/bitnagil/presentation/common/file/ImageFileConverter.kt

```
@@ -0,0 +1,64 @@
+        val mimeType = contentResolver.getType(uri) ?: "image/jpeg"
+
+        val bytes = contentResolver.openInputStream(uri)?.use { it.readBytes() }
+            ?: return@withContext null
+
+        ImageFile(
+            prefix = prefix,
+            name = fileName,
+            mimeType = mimeType,
+            bytes = bytes,
+        )
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)